setup.py

```diff
@@ -77,7 +77,7 @@
             "plotly",
             "matplotlib",
             "sphinx-copybutton",
-            "pydata-sphinx-theme~=0.1",
+            "pydata-sphinx-theme==0.1.1",
```

Comment:
😭😭😭😭😭😭

Reply:
Yep that's fixed it.

Weird that this didn't work:

```
Collecting pydata-sphinx-theme~=0.1
  Downloading pydata_sphinx_theme-0.2.0-py2.py3-none-any.whl (38 kB)
```

Reply:
next release will be v0.2.1 anyway, since I'll be the one that does it tomorrow probably haha

Reply:
yep, I'll wait for that to merge 👍